### PR TITLE
Automate SCRIPTNAME in termux-camera-photo to "${0##*/}"

### PR DESCRIPTION
**Pull request by SDRausty**

### CODE REVIEW

**Comment by SDRausty** [comment]
https://github.com/termux/termux-api-package/issues/146

File: scripts/termux-camera-photo [hunk 1]
```diff
@@ -1,11 +1,10 @@
 #!@TERMUX_PREFIX@/bin/sh
 set -e -u
 
-SCRIPTNAME=termux-camera-photo
 show_usage () {
-    echo "Usage: termux-camera-photo [-c camera-id] output-file"
```
Comment (Grimler91) [inline scripts/termux-camera-photo:6]: Changing from termux-camera-photo to SCRIPTNAME here is good

File: scripts/termux-camera-photo [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
+    echo "Usage: ${0##*/} [-c camera-id] output-file"
     echo "Take a photo and save it to a file in JPEG format."
-    echo "  -c camera-id  ID of the camera to use (see termux-camera-info), default: 0"
```
Comment (Grimler91) [inline scripts/termux-camera-photo:8]: Changing from termux-camera-photo to SCRIPTNAME here is good

**Review by Grimler91 — CHANGES_REQUESTED** [review]
Please update the commit message to something more descriptive. 

The PR also doesn't apply, so please rebase.

**Comment by SDRausty** [comment]
> The PR also doesn't apply, so please rebase.

I have no idea how to do this as the original fork is deleted.

**Pull request closed without merging**